### PR TITLE
update docker

### PR DESCRIPTION
**Pull request by xvjiarui**

### CODE REVIEW

**Comment by hellock** [comment]
Task linked: [CU-4dtkav MMSegmentation](https://app.clickup.com/t/4dtkav)

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ARG PYTORCH="1.5"
+ARG PYTORCH="1.6"
```
Comment (nbei) [inline docker/Dockerfile:1]: It may be "1.6.0" for `dockerhub`. ^_^

**Comment by hellock** [comment]
Is it tested?

**Comment by xvjiarui** [comment]
> Is it tested?

Yes.

**Review by hellock — APPROVED** [review]

**Pull request merged**